### PR TITLE
Pull - Charging section

### PR DESCRIPTION
**Pull request by arpanrau**
Charging section ready for preliminary review

### CODE REVIEW

File: esf_main.tex [hunk 1]
```diff
@@ -853,18 +855,34 @@ \subsubsection{Charging}
...
-	        Charger Type: & TODO \\ \hline
-	        Maximum charging power & TODO W \\ \hline
-	        Maximum charging voltage & TODO V \\ \hline
-	        Maximum charging current & TODO A \\ \hline
-	        Interface with accumulator & TODO \\ \hline
-	        Input voltage & TODO \\ \hline
-	        Input current & TODO \\ \hline
+	        Charger Type: & ELCON PFC 2000+ TCCH-288-6  \\ \hline
+	        Maximum charging power & 24.9 A \\ \hline
+	        Maximum charging voltage & 4.15 V (Per Cell) \\ \hline
```
Comment (byronwasti) [inline esf_main.tex]: I would also include the maximum charging voltage overall (ie. 389V)

File: esf_main.tex [hunk 2]
```diff
@@ -853,18 +855,34 @@ \subsubsection{Charging}
...
+	        Maximum charging current & 6 A \\ \hline
```
Comment (byronwasti) [inline esf_main.tex]: Since we are running it off 115VAC, this is 5A

File: esf_main.tex [hunk 3]
```diff
@@ -853,18 +855,34 @@ \subsubsection{Charging}
...
+	        Interface with accumulator & CAN-Bus \\ \hline
```
Comment (byronwasti) [inline esf_main.tex]: I think this is talking about the high-voltage line, but I could be wrong. @aehoppe or @rileychapman can probably clarify.

**Review by byronwasti — CHANGES_REQUESTED** [review]
A few comments on the table for the charger.

File: esf_main.tex [hunk 4]
```diff
@@ -875,11 +875,11 @@ \subsubsection{Charging}
...
+	The charging system consists of two main components:  A charger that connects to the main pack connector and supplies the current needed to charge the accumulator, and a charging circuit that provides power to the AMS, opens the AIRS if the shutdown button is pressed, and displays critical information while charging. 
 	
-	This CE-Listed charger supplies constant current at 6A until cell voltages reach 4.15V. Then, constant voltage is supplied at 4.15V per cell, summing to 298.8 volts, until the cells are fully charged. The charging power lines ???TODO: Talk to Alex about fusing???. 
+	This CE-Listed charger supplies constant current at 6A until cell voltages reach 4.15V. Then, constant voltage is supplied at 4.15V per cell, summing to 298.8 volts, until the cells are fully charged. The charging power lines are protected via the charger's short-circuit protection, which automatically stops charging if short-circuit is encountered across the output.
```
Comment (aehoppe) [inline esf_main.tex]: "if a short-circuit condition occurs" might be better wording.

File: esf_main.tex [hunk 5]
```diff
@@ -875,11 +875,11 @@ \subsubsection{Charging}
...
 	
-	The charger interfaces with the accumulator through a lever action locking connector from TE Connectivity's HVP 800 series (P/Ns 2141227(accumulator side) and 2141154(charger side). These connectors contain an interlock that opens the AIRS when not engaged to maintain compliance with EV8.3.3. The charger has an interlock function that stops charging when the connector is not engaged, shown in figure \fref{Charger Interlock}. 
+	The charger interfaces with the accumulator through a lever action locking connector from TE Connectivity's HVP 800 series (P/Ns 2141227(accumulator side) and 2141154(charger side)). These connectors contain an interlock that opens the AIRS when not engaged to maintain compliance with EV8.3.3. The charger has an interlock function that stops charging when the connector is not engaged, shown in figure \fref{Charger Interlock}.
```
Comment (aehoppe) [inline esf_main.tex]: 'Opens the shutdown circuit, preventing the AIRs from closing' is the wording I would use. 

Also your `\fref{}` won't work so well if you don't use the fig: prefix in your label like `\label{fig:whatever}`

File: esf_main.tex [hunk 6]
```diff
@@ -891,7 +891,7 @@ \subsubsection{Charging}
 	
 	The accumulator will only be charged on the charging cart. The cart will be able to support the full weight of the accumulator and only move when a dead man's switch is activated.
 	
-	The accumulator cart will contain a specialized shutdown and monitoring circuit that allows the accumulator to be charged through the main pack connector.The shutdown and monitoring circuit will contain a clearly labeled emergency stop push button with a minimum diameter of 25mm which opens the AIRS ands stops charging. It will also communicate with the AMS over CAN and display critical information during charging.
+	The accumulator cart will contain a specialized charging circuit that allows the accumulator to be charged through the main pack connector.The charging circuit will contain a clearly labeled emergency stop push button with a minimum diameter of 25mm which opens the AIRS ands stops charging. It will also communicate with the AMS over CAN and display critical information during charging.
```
Comment (aehoppe) [inline esf_main.tex]: Charging circuit on the cart that mimics the shutdown circuit in the vehicle? Could be a little clearer.

File: esf_main.tex [hunk 7]
```diff
@@ -1162,6 +1162,22 @@ \subsection{Ready to Drive Sound}\label{Ready to Drive}
...
+\begin{figure}[H]
+	\includegraphics[width=\linewidth]{charger_output}
+	\caption{Charger Output Specifications}
+\end{figure}
+
+\begin{figure}[H]
+	\includegraphics[width=\linewidth]{charger_input}
+	\caption{Charger Input and Protection Features} 
+\end{figure}
+
```
Comment (aehoppe) [inline esf_main.tex:1384]: Appendix looks fine to me.

**Review by aehoppe — CHANGES_REQUESTED** [review]

**Pull request merged**